.github/workflows/assign_reviewer.yml

```diff
@@ -0,0 +1,30 @@
+name: PR management
+
+on:
+  pull_request:
+    types: [ready_for_review]
+
+jobs:
+  assign-reviewers:
+    runs-on: ubuntu-latest
+    name: Assign reviewers
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Bootstrap
+        uses: ./.github/actions/bootstrap
+        with:
+          python-version: 3.11
+          poetry-skip: 'true'
+      - name: Authenticate GitHub CLI
+        run: gh auth login --with-token <<< ${{ secrets.GITHUB_TOKEN }}
+      - name: Parse PR labels and assign reviewers
+        run: |
+          python dev/assign_reviewers.py .github/CODEREVIEWERS \
+                                         ${{ github.repository }} \
+                                         ${{ github.event.pull_request.number }} \
+                                         ${{ github.event_path }}
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```

Comment:
Can you do me a favor and test if it works with the [`GH_TOKEN`](https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows) variable. I hope that this will allow us to remove the `Authenticate GitHub CLI` step above. Do we need the token for something else other then the `gh` CLI?

Reply:
I'll update the PR, thanks!